# BTS7960 pin map (BCM numbering)
left_motor:
  RPWM: 12    # PWM
  LPWM: 6
  R_EN: 5
  L_EN: 7
  invert: true

right_motor:
  RPWM: 18    # PWM
  LPWM: 17
  R_EN: 22
  L_EN: 27
  invert: true

pwm:
  freq_hz: 15000   # pigpio 소프트 PWM 주파수(가청대역 밖 권장)
  duty_max: 255    # pigpio 기본 range(0~255)

# 라인센서(디폴트 예시: 좌/중/우 3개, 디지털)
line_sensor:
  left: 23
  center: 24
  right: 25
  active_low: true   # 검은 라인일 때 LOW 출력하는 센서면 true
  pull_up: true

